packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -36,7 +36,7 @@ function ChangesSummary( { revision, previousRevision } ) {
 			data-testid="global-styles-revision-changes"
 			className="edit-site-global-styles-screen-revisions__changes"
 		>
-			{ changes.join( ', ' ) }
+			{ changes.join( ', ' ) }.
```

Comment:
Pretty crude but it was the simplest approach after having tried the following:

- passing a "suffix" option value of `"."` to `getGlobalStylesChanges()`. Rejected because the function returns an array of strings and requires appending the suffix to the last item. Too weird.
- Changing the return value of `getGlobalStylesChanges()` to be a joined string. Rejected because it removes all the flexibility of a `string[]` return value.
- Creating a new function to return `changes.join( ', ' ) }.`. Rejected, or at least postponed, until we need it. We're only using `getGlobalStylesChanges()` in two places.

In summary, I'm of the opinion that the component should decide how to present the changes.

Reply:
Since we're already using `,` to join the changes together, I think appending a `.` character here is likely fine!